activerecord/test/cases/encryption/configurable_test.rb

```diff
@@ -47,22 +47,26 @@ class ActiveRecord::Encryption::ConfigurableTest < ActiveRecord::EncryptionTestC
 
     NamedPirate = Class.new(Pirate) do
       self.table_name = "pirates"
+      self.filter_attributes = []
     end
     NamedPirate.encrypts :catchphrase
 
     assert_includes application.config.filter_parameters, "named_pirate.catchphrase"
+    assert_includes NamedPirate.filter_attributes, "catchphrase"
```

Comment:
I believe we may omit `self.filter_attributes = []`  as `assert_includes` should still pass. Also pre-initializing `filter_attributes` instance variable hides the parent `filter_attributes` issue, I believe if we remove this pre-initialization and add an assertion like `refute_includes ActiveRecord::Base.filter_attributes, "catchphrase"
` it will fail, and we expected `catchphrase` to only exist on the `NamedPirate` class 
```
Expected ["catchphrase"] to not include "catchphrase".
```

```suggestion
    NamedPirate = Class.new(Pirate) do
      self.table_name = "pirates"
      #self.filter_attributes = [] commented out for slightly better git diff
    end
    NamedPirate.encrypts :catchphrase

    assert_includes application.config.filter_parameters, "named_pirate.catchphrase"
    assert_includes NamedPirate.filter_attributes, "catchphrase"
    refute_includes ActiveRecord::Base.filter_attributes, "catchphrase"
```